index.html

```diff
@@ -0,0 +1,195 @@
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <title>Akpuru Solomon Website</title>
+</head>
+
+<body>
+  <div id="container">
+   <header>
+      <div class="topnav" id="myTopnav">
+        <a class="logo" href="#">Akpuru Solomon</a>
+        <a href="#"><img src="./image/Union.png" alt="Message Icon" /></a>
+        <a href="#works" class="active">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+        <a class="bars" href="#"><i class="fa fa-bars"></i></a>
+      </div>
+    </header>
+    <section id="opening">
+      <div id="intro_word">
+      <h1 id="title">Hey There. I’m Solomon</h1>
+      <h2 id="job">I’m a Software Developer</h2>
+      <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+        </div>
+      <ul id="social">
+        <li><a href="https://github.com/Solobarine"><i class="social-icons fa-brands "></i><img src="./image/social_media/github.png" alt="my github handle" width="20" height="20"></a></li>
+        <li><a href="https://www.linkedin.com/in/solomnakpuru/"><i class="social-icons fa-brands "></i><img src="./image/social_media/linkedin.png" alt="my linkedin handle" width="20" height="20" ></a></li>
+        <li><a href="#"><i class="social-icons fa-brands "></i><img src="./image/social_media/angelist.png" alt="my angelist handle" width="20" height="20"></a></li>
+        <li><a href="https://twitter.com/SolomonAkpuru"><i class="social-icons fa-brands "></i><img src="./image/social_media/twitter.png" alt="my twitter handle" width="20" height="20"></a></li>
+        <li><a href=""><i class="social-icons fa-brands fa-microverse"></i><img src="./image/social_media/Vector_m.png" alt="my m handle" width="20" height="20"></a></li>
+      </ul>
+    </section>
+
+    <section id="work">
+      <h2 id="work_title">My Recent Works</h2>
+
+      <div id="projects">
+
+        <div id="frames">
+          <div id="recent-img"><img src="./image/Img_placeholder.png" alt=""></div>
+          <h3 class="stories">Multi-Post Stories</h3>
+          <p class="summary">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="bar">
+            <span>CSS</span>
+            <span>HTML</span>
+            <span>Bootstrap</span>
+            <span>Ruby</span>
```

Comment:
- [ ]  working on the desktop version should not affect your mobile version. I can see that working on the desktop version has affected you mobile design, please fix it. 
Your design  |  Figma design 
![image](https://user-images.githubusercontent.com/43877689/178111836-f565d173-dd92-4306-869b-5a7996713d33.png)  |  ![image](https://user-images.githubusercontent.com/43877689/178111851-bfe2ec71-4ebe-40c5-877a-3c35996edb04.png)